### ISSUE TITLE
[React Native Gallery -> Flyout]: The text inside the colored button is not visible properly in high contrast themes.

### ISSUE THREAD

**Issue by YajurG** [microsoft/react-native-gallery#338]
**Pre-Requisite:**  

Turn on High Contrast Aquatic Theme (Settings->Accessibility->Contrast themes->Aquatic/Dessert theme->Apply)

**Repro-Steps:**

1. Launch the React Native Gallery app.
2. Navigate to ‘Button’ button and invoke it.
3. Observe Colored button is not having proper contrast in high contrast mode

**Actual Result:**

The text inside the button is not clearly visible in high contrast themes.

**Expected Result:**

All the content should be clearly visible in high contrast themes. 

**User impact:**

Users with low vision will not be able to consume the content present in the Flyout card if it is not visible in High Contrast Themes.

Internal bug link: https://microsoft.visualstudio.com/OS/_workitems/edit/45265387
